Gemfile

```diff
@@ -72,9 +72,9 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'rspec-collection_matchers'
   gem 'guard-rspec', require: false
-  gem 'rubocop', require: false
   gem 'pry'
   gem 'pry-byebug'
+  gem 'rubocop', require: false
```

Comment:
Install rubocop globally with `gem install rubocop`. Don't include it in `Gemfile` since it's not a part of this project.